File: spec/python/test_position_in_seq.py
import unittest

from position_in_seq import PositionInSeq

class TestPositionInSeq(unittest.TestCase):
    def test_position_in_seq(self):
        r = PositionInSeq.from_file("src/position_in_seq.bin")

        self.assertEqual(r.numbers, [1, 2, 3])

File: spec/lua/test_position_in_seq.lua
local luaunit = require("luaunit")

require("position_in_seq")

TestPositionInSeq = {}

function TestPositionInSeq:test_position_in_seq()
    local r = PositionInSeq:from_file("src/position_in_seq.bin")

    luaunit.assertEquals(r.numbers, {1, 2, 3})
end
